azure-functions-maven-plugin/pom.xml

```diff
@@ -246,6 +246,8 @@
                 <includes>
                     <include>**/*.xml</include>
                     <include>templates.json</include>
+                    <include>bindings.json</include>
+                    <include>Resources.json</include>

```

Comment:
Is there any reason we capitalize the first letter?

Reply:
These file names are following the pattern of the existing templates files.